bootstrap/hook.go

```diff
@@ -97,12 +105,11 @@ func newHookScriptWrapper(hookPath string) (*hookScriptWrapper, error) {
 			"SET > \"" + h.afterEnvFile.Name() + "\"\n" +
 			"EXIT %" + hookExitStatusEnv + "%"
 	} else {
-		script = "#!/bin/bash\n" +
```

Comment:
I ditched this shebang, we always invoke these scripts with an interpreter and it was causing problems on windows bash.